scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1025,56 +1026,71 @@ static bool match_one(struct layer_match *match, struct task_struct *p, const ch
 	}
 }
 
-static bool match_layer(struct layer *layer, struct task_struct *p, const char *cgrp_path)
+int match_layer(u32 layer_id, pid_t pid, const char *cgrp_path)
```

Comment:
For global functions I couldn't remember if bool return values were allowed, so I switched this to return an int as IIRC that was supported in the initial implementation.